envrec/env.go

```diff
@@ -0,0 +1,105 @@
+package envrec
```

Comment:
*not used yet*

I implemented this package accordingly to the specification... but then I realized it was a super super high level concept and the types.Object type could be used instead.
I'll leave this package here until we can confirm this is bullshit.